### PR TITLE
Add login page with validation form

### PR DESCRIPTION
**Pull request by vadymchornyidev**
> Create a page for login
> Supplement an authentication service (login user)
> Add route
> Create buttons for remembering users and for rerouting to forgot password

🔥🔥🔥

### CODE REVIEW

File: src/components/authentication/login/index.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+function Login() {
+    let history = useHistory();
+
+    const onFinish = (values) => {
+        login(values, history);
+    };
+
+    function onChange(e) {
+        console.log(`remember me = ${e.target.checked}`);
```
Comment (marynakernychna) [inline src/components/authentication/login/index.jsx]: A function that in fact does nothing except displaying a message in the console when the status of the check box changes. What is it for?

**Review by marynakernychna — CHANGES_REQUESTED** [review]
I also advise you to delete many commas where they are not needed

File: src/components/authentication/login/index.jsx (hunk 1 above)
Comment (vladtymo) [reply src/components/authentication/login/index.jsx]: @VadimChorrny look through the code and remove all console.log's.

File: src/components/authentication/styles.module.css [hunk 2]
```diff
@@ -99,13 +99,23 @@
...
 }
 
+.helperForm {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+}
+
+.checkboxForm {
+    color: #fff !important;
```
Comment (vladtymo) [inline src/components/authentication/styles.module.css:116]: Try not to use the !important property

File: src/services/authentication.js [hunk 3]
```diff
@@ -1,25 +1,68 @@
...
+    authenticationService
+        .registerUser(model)
+        .then(
+            () => {
+                history.push("/login");
+            },
+            (err) => {
+                err.response.status == 400
+                    ? AlertService.errorMessage(
+                          "Registration failed",
```
Comment (vladtymo) [inline src/services/authentication.js]: Consider moving these messages to constants.

File: src/components/authentication/registration/index.jsx [hunk 4]
```diff
@@ -111,40 +115,43 @@ function Registration() {
                             },
                             ({ getFieldValue }) => ({
                                 validator(_, value) {
-                                    if (!value || getFieldValue('password') === value) {
+                                    if (
+                                        !value ||
+                                        getFieldValue("password") === value
+                                    ) {
```
Comment (YevheniiVovk) [inline src/components/authentication/registration/index.jsx:127]: Let's change to 
 if (!value 
     || getFieldValue('password') === value) {

**Review by vladtymo — APPROVED** [review]

**Review by marynakernychna — DISMISSED** [review]

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**